docs/examples/hairpin_fitting/hairpin_unfolding.rst

```diff
@@ -0,0 +1,289 @@
+
+Hairpin unfolding
+=================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+.. _hairpin_fitting:
+
+Analyzing fd curve with hairpin unfolding event
+-----------------------------------------------
+
+In this notebook we will analyze a force-extension curve of a construct with two DNA handles with a DNA hairpin in between. The hairpin unfolds as the force on the construct is increased.
+
+First, we will compute the high frequency distance, also called :ref:`piezo distance <piezo_tracking>`. Then, we will use the Worm-like chain model to extract the contour length of the unfolded hairpin.
```

Comment:
Do you see why the link is dead?

Reply:
Because the label doesn't exist. You have to make a label to reference it. You can put the label where you want it by adding the following to that file:
`.. _piezo_tracking:`